projects/ngx-device-detector/src/lib/device-detector.service.ts

```diff
@@ -111,6 +115,13 @@ export class DeviceDetectorService {
       device: this.device,
       os_version: this.os_version,
       browser_version: this.browser_version,
+      deviceType: this.isTablet()
+        ? 'tablet'
```

Comment:
Instead of using hardcoded strings, let's create an enum:
```typescript
export enum DeviceType {
  Mobile = 'mobile',
  Tablet = 'tablet',
  Desktop: 'desktop'
}
```

Reply:
Added it